.gitignore

```diff
@@ -9,7 +9,7 @@ node_modules
 .eslintcache
 
 yarn-error.log
-build
+/build
```

Comment:
will this be relative to the project?

Reply:
[Yes]. Otherwise `ci/build` would be ignored.

Reply:
Ah I see, good catch.